deployer/deployer-services/src/main/java/io/hyscale/deployer/services/model/PodStatusCode.java

```diff
@@ -19,20 +19,19 @@
 import java.util.Map;
 
 public class PodStatusCode {
-    
     private PodStatusCode() {}
-
-	private static final Map<Integer, Signals> statusCodeVsMessage = new HashMap<>();
-
-	static {
-		for (Signals each : Signals.values()) {
-			statusCodeVsMessage.put(each.getCode(), each);
-		}
-	}
-
+    
```

Comment:
So what I meant is to remove this class and just rely on the Signals enum. The method which fetches code should be indicative.